### ISSUE TITLE
Idea: Add option for quick exit on error

### ISSUE THREAD

**Issue by gowerc** [gowerc/diffdf#25]
@kieranjmartin ,

Am thinking in the case of super large datasets it might be preferable to have the option to abort the comparison after detecting the first variable with differences. As sometimes users just need a starting point of where to fix rather than finding every single issues. 

In addition it then might be worth then adding an option to specify the order in which variables are checked i.e.    c("LBTESTCD",  "AVAL", "AVALU")  etc     (Can be partial just 3 variables with all the others being auto filled in alphabetical order or something ?

**Comment by kieranjmartin** [comment]
Yes, this might be good. I think doing this might be quite challenging from a programming point of view. This probably does integrate with rcpp quite well as it loops over differences, making early halts a bit easier
